webextensions/api/topSites.json

```
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/topSites",
           "support": {
             "chrome": {
-              "version_added": "≤58"
+              "version_added": "20"
```

:x: https://github.com/chromium/chromium/commit/641bdeb0830b56bd715699126176cc750c10fb92 landed in [19.0.1078.0](https://github.com/chromium/chromium/releases/tag/19.0.1078.0) before 19.0.1084.

```suggestion
              "version_added": "19"
```